R-packages/covidcast/R/covidcast.R

```diff
@@ -204,7 +204,7 @@ COVIDCAST_BASE_URL <- 'https://api.covidcast.cmu.edu/epidata/api.php'
 #' @importFrom dplyr %>%
 covidcast_signal <- function(data_source, signal,
                              start_day = NULL, end_day = NULL,
```

Comment:
this change should be removed. If you reinstall `covidcast` it loads 0.4.1. The new options are
```r
geo_type = c("county", "hrr", "msa", "dma", "state", "hhs", "nation")
```

Reply:
Unfortunately, this branch has gotten too far behind `main`.

Reply:
Fixed.